### DISCUSSION TITLE
Stacked Area Charts by Group

### DISCUSSION THREAD

**Discussion by CarlosValdiviaTorre** [apexcharts/apexcharts.js#4072] [Ideas]
Hello everyone,

First of all, thank you for your work on this wonderful javascript library with which we can make such good graphics.

The reason for this thread is to know the reason why it is possible to stack series by groups in bar charts but not in area charts. Is it something you plan to change in the future?

Thank you very much again and regards

**Comment by rosco54** [comment]
I'm not sure if it matches your requirement exactly but the next released version (after 3.48.0) will be able to stack both bar and area/line series. Series that are referenced by/assigned to the same yaxis will be stacked when ```chart.stacked: true```. Different series types would usually be assigned to different Y axes. Different groups of area/line series are not working yet due to a bug that stacks all series of that type together. It should be stacking separately on different Y axes.

Here's a deliberately overly cluttered example shapshot of a chart that was used for testing purposes:
https://github.com/apexcharts/apexcharts.js/blob/main/tests/e2e/snapshots/mixed/line-column-area-stacked-grouped.png

Here's a HTML source file that will create that chart:
https://github.com/apexcharts/apexcharts.js/blob/main/samples/vanilla-js/mixed/line-column-area-stacked-grouped.html
